torch/csrc/Module.cpp

```diff
@@ -431,15 +434,21 @@ PyObject *THPModule_addDocStr(PyObject *_unused, PyObject *args)
   if (Py_TYPE(obj) == &PyCFunction_Type) {
     PyCFunctionObject* f = (PyCFunctionObject *)obj;
     if (f->m_ml->ml_doc) {
-      return PyErr_Format(PyExc_RuntimeError,
-          "function '%s' already has a docstring", f->m_ml->ml_name);
+      // temporary workaround
```

Comment:
add a comment describing what the temporary workaround is and for what.

Reply:
Ooops. This should be removed.